src/ejercicios.js

```diff
@@ -2,34 +2,57 @@ const db = new BaseDatos();
 db.connect();
 
 function leerDesdeDB() {
-  Utils.disableAll();
-  const id = prompt('ID a leer:');
-  const data = db.read(id);
-  Utils.enableAll();
-  return data;
+  try {
+    Utils.disableAll();
+    const id = prompt('ID a leer:');
+    const data = db.read(id);
+    return data;
+  } catch (err) {
+    console.info(err);
+    alert(`Error!\n\n\t${err.message}`);
+    return null;
+  } finally {
+    Utils.enableAll();
+  }
 }
 
 function btnLeer() {
   const data = leerDesdeDB();
   console.info('Resultado:', data);
 }
 
-// function btnConectar() {
-//   Utils.disableAll();
-//   db.connect();
-//   Utils.enableAll();
-// }
+function btnConectar() {
+} try {
```

Comment:
El bloque `try` debe estar dentro de la `function`
```suggestion
  try {
```